osidb/models.py

```diff
@@ -1506,6 +1511,7 @@ def _create_flaw(self, *args, **kwargs) -> Flaw:
 
         # Flaw model has to be created first
         model, data = [i for i in main_model.items()][0]
+        print(data)
```

Comment:
This seems like forgotten temporary print :smiley:

Reply:
oops yes 😅